README.md

```diff
@@ -382,7 +382,7 @@ _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from
 1. The module operator manifests from the `default` kustomization (not the controller image) will now be bundled and pushed.
    Assuming the settings from [Prepare and build module operator image](#prepare-and-build-module-operator-image) for single-cluster mode, and assuming the following module settings:
    * hosted at `op-kcp-registry.localhost:8888/unsigned`
-   * generated for channel `regular`
+   * generated for channel `regular` (or any other meaningful channel name)
```

Comment:
Personally, I still wouldn't know that the channel name may be arbitrary. I can't see this mentioned in the [docu linked in the beginning of this README](https://github.com/kyma-project/community/tree/main/concepts/modularization#release-channels). If it's mentioned somewhere else, perhaps we should add a link here?
My suggestion would be:
```generated for channel `regular` (you can use another name, [see here](some.link))```

Reply:
agree, I'll edit late.